src/styles/normalizes.scss

```diff
@@ -0,0 +1,16 @@
+:root {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
```

Comment:
It's better apply to the body or html element, :root in most cases is used for css variables. Margin here is make no sense